### DISCUSSION TITLE
add dark mode to the official npm website

### DISCUSSION THREAD

**Discussion by KevinRebakure** [community/community#139019] [npm]
I personally prefer to keep all my working environments in dark mode and whenever I go to a site without a dark mode feature. I just can't take it, it bothers me a lot. 

I would like us to that feature on the official npm website, because personally it's my go to, for looking up documentations fast for packages I need.
 
I'm not alone though😅

**Comment by tshandler** [comment]
I have two solutions to make the bright website dark.

1. Traditional Method
As you know, we can add some of feature to make your bright website dark into CSS and HTML.

2. Modern Method
If you are using modern JavaScript framework or library, you can use pre-built node modules such as user-dark-mode and react-dark-mode-toggle.

You are able to choose one of them and one of the modules for your website.
Hoping this article to make you assist.

**Reply by TrustworthyDev** [reply]
It's helpful to set dark mode to the official website.

**Reply by KevinRebakure** [reply]
Yeah, I agree. It's really a basic feature present in most of all documentations...

**Comment by tshandler** [comment]
What method did you use for your dark mode?

**Reply by KevinRebakure** [reply]
I didn't invest a significant amount of time trying to come up with a cool dark mode. I settled for this normal [dark mode](https://chromewebstore.google.com/detail/dark-mode/dmghijelimhndkbmpgbldicpogfkceaj?hl=en-US&utm_source=ext_sidebar) chrome extension. It doesn't look that good on npm, but atleast the default bright light is gone😅. I'm settling for that in the meantime. If they ever give us the dark mode feature...

**Reply by aryomuzakki** [reply]
use the [dark reader](https://chromewebstore.google.com/detail/dark-reader/eimadpbcbfnmbkopoojfekhnkhdbieeh) since long ago, 


I only now looking at [this feature request](https://github.com/orgs/community/discussions/128400) that suprisingly exist since years ago :(

**Reply by KevinRebakure** [reply]
Guess that's a nice hack too

**Reply by codeXafzal** [reply]
> use the [dark reader](https://chromewebstore.google.com/detail/dark-reader/eimadpbcbfnmbkopoojfekhnkhdbieeh) since long ago,
> 
> I only now looking at [this feature request](https://github.com/orgs/community/discussions/128400) that suprisingly exist since years ago :(

you  can use flags and force black theme its nice if you don't want white light though

**Comment by tshandler** [comment]
Fingers crossed that your choice works out for you!

**Comment by gameroman** [comment]
❤️❤️❤️ THANK YOU ❤️❤️❤️

**Comment by tj-commits** [comment]
THIS HAS BEEN ADDED
